README.md

```diff
@@ -81,30 +81,30 @@ You can call any method on authenticated user. For example, let's send message t
   var result = await client.GetContactsAsync();
 
   //find recipient in contacts
-  var user = result.Users.lists
+  var user = result.Users
 	  .Where(x => x.GetType() == typeof (TLUser))
 	  .Cast<TLUser>()
-	  .FirstOrDefault(x => x.phone == "<recipient_phone>");
+	  .FirstOrDefault(x => x.Phone == "<recipient_phone>");
 	
   //send message
-  await client.SendMessageAsync(new TLInputPeerUser() {user_id = user.id}, "OUR_MESSAGE");
+  await client.SendMessageAsync(new TLInputPeerUser() {UserId = user.Id}, "OUR_MESSAGE");
 ```
 
 Full code you can see at [SendMessage test](https://github.com/sochix/TLSharp/blob/master/TLSharp.Tests/TLSharpTests.cs#L87)
 
 To send message to channel you could use the following code:
 ```csharp
   //get user dialogs
-  var dialogs = await client.GetUserDialogsAsync();
+  var dialogs = (TLDialogs) await client.GetUserDialogsAsync();
```

Comment:
this change seems unrelated to PascalCase API?

Reply:
no, but if you remove this cast the "dialogs.Chats" gets compile error

Reply:
I'm not talking about dialogs.Chats here

Reply:
i edit readme file so a user can copy and paste snippets into their project and successfully compile it, if this cast wheren't here code cause compile error

Reply:
what would be the compiler error here?

Reply:
`GetUserDialogsAsync` function returns `TLAbsDialogs` object and is a parent class that `TLDialogs` inheritance from it. `TLAbsDialogs` dosen't have a member called `Chats`, so you should cast return value of `GetUserDialogsAsync` to child class of `TLDialogs` that dose have a member called `Chats`

Reply:
makes complete sense, thanks for your patience replying to my questions! merging now